lib/super_good/solidus_taxjar/api_params.rb

```diff
@@ -37,13 +37,13 @@ def tax_rate_address_params(address)
           }.merge(order_address_params(address))
         end
 
-        def transaction_params(order)
+        def transaction_params(order, transaction_id = order.number)
           {}
             .merge(customer_params(order))
             .merge(order_address_params(order.tax_address))
             .merge(transaction_line_items_params(order.line_items))
             .merge(
-              transaction_id: order.number,
+              transaction_id: transaction_id,
```

Comment:
Added a comment to #101 around updating this to be the orders latest/current transaction ID once the work to associate an order with taxjar transactions has been completed.